tox-requirements.txt

```diff
@@ -1,2 +1,2 @@
-docker-compose==1.9.0
+#docker-compose>=1.9.0
```

Comment:
i couldnt get this to install successfully anymore and I don't think we actually use it with tox :/

Reply:
what in the world, i thought it would use supervisord at most

is this an artifact of a different way to run docker on EC2?

Reply:
this was when trying to set up a venv locally, this is just used for tests i believe